commcare_connect/opportunity/models.py

```diff
@@ -70,3 +70,18 @@ class DeliverForm(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class OpportunityAccess(models.Model):
```

Comment:
What happened to the other fields on this model? I know we mentioned not creating ones that referenced uncreated models, but some of the fields in the spec were specfic to this one and are still missing. Is that intentional?

Reply:
`date_claimed` is the one I was thinking of. Agree that the others are not necessary yet.

Reply:
Added. [4ad1eaf](https://github.com/dimagi/commcare-connect/pull/41/commits/4ad1eaf0cda305b5fbe0d1cde7ecfd476fcbbf52)